crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -833,13 +861,38 @@ type SystemParameters {
 }
 
 type TransactionBlock {
+	"""
+	The transaction block effects provide information such as gas cost and object changes
+	"""
 	effects: TransactionBlockEffects
+	"""
+	Sender of the transaction block
+	"""
 	sender: Address
+	"""
+	The raw transaction block in BCS format
+	"""
 	bcs: Base64
+	"""
+	The gas input of the transaction block
+	"""
 	gasInput: GasInput
 	kind: TransactionBlockKind
+	"""
+	A list of signatures signed by all transaction participants.
```

Comment:
Who is a participant of a transaction? What is a "non-participant signature"? The wording on the comment about non-participant signatures is also a little strange -- it's phrased as a kind of command, but IIUC, it's talking about something that the API guarantees?